package.json

```diff
@@ -65,31 +65,15 @@
 			"tsx"
 		],
 		"rules": {
+			"import/no-unresolved": "off",
 			"import/no-unassigned-import": "off",
-			"no-unused-vars": [
-				2,
-				{
-					"varsIgnorePattern": "^React$"
```

Comment:
Yes, https://github.com/xojs/eslint-config-xo-typescript/blob/2604fb2f18c7e268c7117c0cadf06faac4c81098/index.js#L144

Reply:
Why is it just in the typescript config and not xo config?

Reply:
I dunno. You're right, it doesn't belong in that config. Reverted.

Reply:
I mean it could be in the shared config, just not only the typescript one. Does `@typescript-eslint/no-unused-vars` supersede the regular `eslint/no-unused-vars`?